runbotics/runbotics-orchestrator-ui/src/src-app/store/slices/Credentials/Credentials.thunks.ts

```diff
@@ -31,3 +31,6 @@ export const deleteCredential = ApiTenantResource
 
 export const updateAttribute = ApiTenantResource
     .patch<BasicCredentialDto, EditAtributeDto>(':credentialId/UpdateAttribute/:attributeName', CREDENTIAL_PATH);
+
+export const fetchAllCredentialsAssignedToProcess = ApiTenantResource
+    .get<BasicCredentialDto[]>(':processId', 'process-credentials/processes');
```

Comment:
Here the return type is actually formatted AvailableCredentials by formatProcessCredentials, so maybe that's why action.payload in extra reducers causes error and ts-ignore is needed, because you assign to allAvailable BasicCredentialDto[] where you defined in state that it should be AvailableCredentials[].
Again - happy to discuss if I am wrong, cause maybe I am missing something

Reply:
You're right, changed types, removed ts-ignore